src/docs/product/data-management-settings/dynamic-sampling/sampling-types.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Sampling Types
+sidebar_order: 20
+description: "Learn more about the types of sampling available for your project."
```

Comment:
```suggestion
description: "Learn about the sampling types and benefits available for your project"
```

Reply:
Learn more about.... is the preferred expression (from what I saw).

Reply:
Either is fine